commit f6c6dd3dde556f2a55b69f17b2053a19f8492b2d (tag: v1.2.0, refs/changes/83/2483/3)
Author: David Pacheco <dap@joyent.com>
Date:   2017-08-31T13:32:15-07:00 (2 years, 1 month ago)
    
    joyent/mdb_v8#93 release v1.2.0
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 15b5496..2106196 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright (c) 2017, Joyent, Inc.
 -->
 
 # mdb_v8 changelog
@@ -14,6 +14,13 @@
 
 None.
 
+## v1.2.0 (2017-08-31)
+
+* #44 want "jsfunction" dcmd
+* #88 want better support for bound functions
+* #91 CTRL+C of ::findjsobjects, followed by ::findjsobjects reports only some objects
+* #85 missing dependency for mdb\_v8\_version.c leads to build failure
+
 ## v1.1.4 (2017-01-30)
 
 * #77 Node v6.6.0 issues
diff --git a/README.md b/README.md
index aabf8b5..27642a9 100644
--- a/README.md
+++ b/README.md
@@ -44,17 +44,17 @@ https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8.  If you have the
 can list the latest binaries with:
 
     $ mfind -t o $(mget -q /Joyent_Dev/public/mdb_v8/latest)
-    /Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_amd64.so
-    /Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_ia32.so
+    /Joyent_Dev/public/mdb_v8/v1.2.0/mdb_v8_amd64.so
+    /Joyent_Dev/public/mdb_v8/v1.2.0/mdb_v8_ia32.so
 
 You can fetch a specific binary like this (in this case, the 32-bit version
-1.1.4 binary):
+1.2.0 binary):
 
-    $ mget -O /Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_ia32.so
+    $ mget -O /Joyent_Dev/public/mdb_v8/v1.2.0/mdb_v8_ia32.so
 
 or using curl:
 
-    $ curl -O https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8/v1.1.4/mdb_v8_ia32.so
+    $ curl -O https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8/v1.2.0/mdb_v8_ia32.so
 
 This one-liner will get you the latest 32-bit binary:
 
diff --git a/version b/version
index 5f84e78..9a05c1a 100644
--- a/version
+++ b/version
@@ -8,4 +8,4 @@
 #
 # This value is also used by the "publish" target in the build process.
 #
-1.1.5
+1.2.0
